src/props/props.js

```diff
@@ -157,7 +157,7 @@ export type Props = {|
     merchantID : $ReadOnlyArray<string>,
     enableOrdersApprovalSmartWallet : boolean | void,
     smartWalletOrderID : string | void,
-    disableSetCookie : boolean
+    disableSetCookie? : boolean
```

Comment:
Let's have this always be a boolean and default the value to false similar to `vault` prop
```suggestion
    disableSetCookie : boolean
```